userguide/dailyuse/waydroid.rst

```diff
@@ -0,0 +1,83 @@
+Android apps
+========================
+
+`Waydroid <https://waydro.id>`_ is a minimal Android container and compatibility layer to run Android apps on GNU/Linux operating systems such as Ubuntu Touch.
+
+.. note::
+    "Computer" refers to another device you connect your Ubuntu Touch device to (via USB here).
+    Your USB-attached computer must have ``adb`` and ``fastboot`` installed.
+
+.. note::
+    You will need to execute commands on your Ubuntu Touch device (and/or attached computer) to initialize Waydroid and install APKs.
+    You can do that by using the terminal application, but it is easier to use ``adb shell`` or :doc:`set up SSH </userguide/advanceduse/ssh>` to access your Ubuntu Touch device from your computer.
+
+
+How to set up
+--------------
+
+All Halium 9 and newer devices ship with waydroid out of the box starting Ubuntu Touch focal, if your device fits these requirements you can follow the folowing steps
+
+#. Run ``adb shell`` from your computer to get a shell on your Ubuntu Touch device (or open the terminal app on your ubuntu touch device).
+#. Run the following command on your Ubuntu Touch device: ``waydroid init``.
+
+Now you're done! You might have to refresh the app drawer (pull down from the center of the screen and release) for the new Android apps to show up.
+
+.. note::
+    You now have an ADB server running on your Ubuntu Touch device.
+
+How to install new APKs
+-----------------------
+
+- Copy the APK to ``/home/phablet/Downloads``. Then run the following from your computer::
+
+    adb shell # connect from your computer to your Ubuntu Touch device you can also simply open de terminal app on your Ubuntu Touch device
+    waydroid app install /home/phablet/Downloads/my-app.apk #
+    exit
+
+- Done! You might have to refresh the apps scope (pull down from the center of the screen and release) for the new Android apps to show up.
+
+Keep your apps up to date
+-------------------------
+
+- To keep your apps up to date you can use of F-Droid or ApkTrack. If you want to install any of the above apps you can find them here:
+
+- F-Droid: https://f-droid.org/
+- ApkTrack: https://f-droid.org/packages/fr.kwiatkowski.ApkTrack/
+
+How to uninstall apps
+---------------------
+
+- To uninstall apps from the Ubuntu Touch device, run ``adb uninstall [APP_ID]`` from your computer::
+
+    adb shell # connect from your computer to your Ubuntu Touch device, you can also simply open de terminal app on your Ubuntu Touch device
+    waydroid app remove [APP_ID] # you can find the APP_ID by using waydroid app list
```

Comment:
Maybe put `waydroid app list` in quotes?

Reply:
fixed